docs/topics/auth/default.txt

```diff
@@ -487,6 +487,40 @@ The login_required decorator
 
     The login_required decorator does NOT check the is_active flag on a user.
 
+.. currentmodule:: django.contrib.admin.views.decorators
+
+The staff_member_required decorator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. function:: staff_member_required([redirect_field_name=REDIRECT_FIELD_NAME, login_url='admin:login'])
+
+    As a shortcut, you can use
+    :func:`~django.contrib.admin.views.decorators.staff_member_required` decorator::
+
+        from django.contrib.admin.views.decorators import staff_member_required
+
+        @staff_member_required
+        def my_view(request):
+            ...
+
+    :func:`~django.contrib.admin.views.decorators.staff_member_required` does
+    the following:
+
+    * If the user is logged in, is a staff member (meaning that user
+      object's property ``is_staff=True``), and is active
+      (``is_active=True``), execute the view normally.
+
+    * Otherwise, redirect to :setting:`settings.LOGIN_URL
+      <LOGIN_URL>`, passing the current absolute path in the query string.
+      Example: ``/accounts/login/?next=/polls/3/``.
+
+    :func:`~django.contrib.admin.views.decorators.staff_member_required` takes
```

Comment:
include this description in the reference doc
keep the topic guide more focused on usage.